spec/tasks/uploads_spec.rb

```diff
@@ -216,4 +216,24 @@ def invoke_task
       )
     end
   end
+
+  describe "uploads:downsize" do
```

Comment:
that doesn't test much compared to all the cases of the task

Reply:
Yeah. tbh I added this specific spec just because I have another PR that fixes an issue with the script and wanted to add a regression test 😅 so this PR is providing a way to test this, and adds a positive test case 😃 So, a good starting point. Hopefully there will be time for expanding the spec suite a bit more